### DISCUSSION TITLE
Nginx Configuration

### DISCUSSION THREAD

**Discussion by MustafaBulu** [apache/superset#18306] [Q&A / Help]
I set the gunicorn to superset as it says in the doc. (https://superset.apache.org/docs/installation/configuring-superset)

![supset](https://user-images.githubusercontent.com/78935007/123109476-f304af80-d443-11eb-869b-f43e14477155.png)

But I couldn't configure nginx. How should I configure Nginx?

**Comment by zufolo441** [comment]
our configuration:
superset server is 192.168.1.80

server {
	listen 159.213.63.16:80;
	server_name dwh.istitutodegliinnocenti.it;
        include /etc/nginx/acmechallenge.conf;
}
server {
	listen 159.213.63.16:443 ssl;
	server_name dwh.istitutodegliinnocenti.it;

	root /var/www/html;

	index index.html index.htm index.nginx-debian.html;
    
    	ssl_certificate /etc/ssl/dwh.istitutodegliinnocenti.it/domain.crt;
	ssl_certificate_key /etc/ssl/dwh.istitutodegliinnocenti.it/domain.key;

	ssl_protocols TLSv1.2 TLSv1.3;
	ssl_prefer_server_ciphers on; 
	ssl_dhparam /etc/nginx/dhparam.pem;
	ssl_ciphers ECDHE-RSA-AES256-GCM-SHA512:DHE-RSA-AES256-GCM-SHA512:ECDHE-RSA-AES256-GCM-SHA384:DHE-RSA-AES256-GCM-SHA384:ECDHE-RSA-AES256-SHA384;
	ssl_ecdh_curve secp384r1;
	ssl_session_timeout  10m;
	ssl_session_cache shared:SSL:10m;
	resolver localhost valid=300s;
	resolver_timeout 5s; 

	if ($request_uri ~ ^(.*)\?$) { return 301 $1; }
	access_log  /var/log/nginx/dwh_access.log;
	error_log  /var/log/nginx/dwh_error.log;
	location / {
        	proxy_pass http://192.168.1.80:8089;
	        proxy_set_header Host $host;
        	proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
	        proxy_redirect off;
	}
}
